airbyte-cdk/java/airbyte-cdk/typing-deduping/src/main/kotlin/io/airbyte/integrations/base/destination/operation/AbstractStreamOperation.kt

```diff
@@ -133,7 +133,17 @@ abstract class AbstractStreamOperation<DestinationState : MinimumDestinationStat
     private fun prepareStageForTruncate(
         destinationInitialStatus: DestinationInitialStatus<DestinationState>,
         stream: StreamConfig
-    ) {
+    ): Pair<InitialRawTableStatus, String> {
```

Comment:
do we ever return something else than the input parameter as a 1st element of the pair?

Reply:
we don't return the entire input param, it's either the initialRawTableState or initialTempRawTableState

Reply:
oh yeah, I didn't see the different return values. Those names are not ideal. We need to think of something better for the future

Reply:
we flip between using tmpTableStatus vs actualRawTableStatus I think.